source/extensions/filters/http/cache/cache_headers_utils.h

```diff
@@ -96,6 +93,10 @@ class CacheHeadersUtils {
   // header_entry is null or malformed.
   static SystemTime httpTime(const Http::HeaderEntry* header_entry);
 
+  // Calculates the age of a cached response
+  static Seconds calculateAge(const Http::ResponseHeaderMap& response_headers,
+                              const SystemTime response_time, const SystemTime now);
```

Comment:
Prefer to not make value parameters const as it doesn't actually do anything (for function declarations at least)

Reply:
I agree. `const` modifier removed from the declaration.